osquery/tables/system/linux/processes.cpp

```diff
@@ -380,6 +423,16 @@ void genProcess(const std::string& pid, QueryData& results) {
   r["system_time"] = proc_stat.system_time;
   r["start_time"] = proc_stat.start_time;
 
+  if (!proc_io.status.ok()) {
+    // /proc/<pid>/io can require root to access, so don't fail if we can't
+    VLOG(1) << proc_io.status.getMessage();
+  } else {
+    r["disk_bytes_read"] = proc_io.read_bytes;
+    r["disk_bytes_written"] =
+        std::to_string(std::stol(proc_io.write_bytes, nullptr) -
```

Comment:
Can you use osquery::safeStrtol

Reply:
Yep. Looks like this is merged, so I'll put up another PR with it.